src/sqlparser/src/parser.rs

```diff
@@ -3229,11 +3251,11 @@ impl Parser<'_> {
                     deferred,
                 }
             } else if materialized
-                && let Some(rate_limit) = self.parse_alter_streaming_rate_limit()?
+                && let Some(rate_limit) = self.parse_alter_backfill_rate_limit()?
             {
-                AlterViewOperation::SetStreamingRateLimit { rate_limit }
+                AlterViewOperation::SetBackfillRateLimit { rate_limit }
             } else {
-                return self.expected("SCHEMA/PARALLELISM/STREAMING_RATE_LIMIT after SET");
+                return self.expected("SCHEMA/PARALLELISM/BACKFILL_RATE_LIMIT after SET");
```

Comment:
Why is it unable to change `STREAMING_RATE_LIMIT` for a streaming job after this PR?

Reply:
Streaming jobs only have `backfill_rate_limit`.

It doesn't make sense to have a `streaming_rate_limit` after backfilling completes. Because we can't rate limit a stream for the running stream job, otherwise the barrier latency will spike according to the rate limit period.